scripts/src/fodt/fix_ignored.py

```diff
@@ -0,0 +1,166 @@
+import logging
+import re
+import requests
+# import shutil
+# import tempfile
+
+from pathlib import Path
+
+import click
+
+from fodt.constants import ClickOptions
+
+class FixIgnored:
+    def __init__(self, maindir: str) -> None:
+        self.maindir = maindir
+
+
+    # Examples of phrases:
+    # --------------------
+    # APIVD: "This keyword is ignored by OPM Flow and has no effect on the simulation <text:span text:style-name="T5812">but is documented here for completeness.
+    #
+    def fix_file(self, file: Path, critical_keywords: set[str]) -> None:
+        keyword = file.stem
+        critical = keyword in critical_keywords
+        # NOTE: Since the files are already backed up by git, we do not create a backup here
+        # Take backup of the file to /tmp
+        #tempfile_ = tempfile.mktemp()
+        #shutil.copy(str(file), tempfile_)
+        line_no = 0
+        lines = []
+        changed = False
+        with open(file, "r") as f:
+            # Read file line by line
+            while line := f.readline():
+                line_no += 1
+                # NOTE: Use re.DOTALL to make "." match the trailing newline
+                if match := re.match(r"""(\s*<text:p\s+.*?>)(.*?)(</text:p>.*$)""", line, re.DOTALL):
+                    start, txt, end = match.groups()
+                    # Remove span tags from txt
+                    txt = re.sub(r"""<text:span .*?>(.*?)</text:span>""", r"\1", txt)
+                    txt = re.sub(r"""<text:span>(.*?)</text:span>""", r"\1", txt)
+                    if "is ignored by OPM Flow and has no effect on the simulation" in txt:
+                        logging.info(f"Found ignored match in {file}:{line_no}.")
+                        txt = self.replace_ignored(txt, critical)
+                        line = start + txt + end
+                        changed = True
```

Comment:
Just one question. There is ugly case with `... simulatio</text:span><text:span text:style-name="T2789">n.</text:span>`. That is probably not covered and needs to be done manually. I guess that is ok, though as we cannot anticipate these kind of formatting things.

Reply:
> That is probably not covered and needs to be done manually.

@blattms If you mean the case that there is no space between closing and starting tag: `</text:span><text:span>`, it should also work. I tested it now.